### PR TITLE
Add haproxy configuration doing like F5

### PR DESCRIPTION
**Pull request by Jooho**
Add haproxy configuration doing like F5. The haproxy in this doc stand for software load balancer in front of multiple masters. This configuration is verified by client.

### CODE REVIEW

File: playbooks/operationalizing/cloudforms_networking.adoc [hunk 1]
```diff
@@ -55,6 +55,28 @@ Also configured in the LTM are two sets of load balancing pools. Load balancing
 
 Once these two components have been defined, the virtual server associated to port 8443 can include the load balancing pool containing the set of master instances on port 8443. The virtual server associated to port 5000 can include the load balancing pool containing the set of node instance that contain running instances of the integrated HAProxy router to pass traffic to the Hawkular metrics services within the OpenShift cluster.
 
+===== Haproxy Configuration
```
Comment (sabre1041) [inline playbooks/operationalizing/cloudforms_networking.adoc]: The formal spelling of HAProxy should be used where possible

File: playbooks/operationalizing/cloudforms_networking.adoc [hunk 2]
```diff
@@ -55,6 +55,28 @@ Also configured in the LTM are two sets of load balancing pools. Load balancing
...
+Software load balancer(haproxy) can be used which Openshift provide if there is no F5. In this case, Haproxy play a role in both load balancer for multiple masters and a reverse  ↳proxy for hawkular metrics. Following haproxy configuration is for hawkular metrics. (It should be added at the end of /etc/haproxy/haproxy.cfg file)
```
Comment (sabre1041) [inline playbooks/operationalizing/cloudforms_networking.adoc]: * Need a space after balancer. 
* There is an  ↳ in line 
* Please add a "The" before "Following haproxy configuration.."

File: playbooks/operationalizing/cloudforms_networking.adoc [hunk 3]
```diff
@@ -55,6 +55,28 @@ Also configured in the LTM are two sets of load balancing pools. Load balancing
...
+backend hawkular-metrics
+    balance source
+    mode tcp
+    option tcplog
+
+# Use physical ip for infra node
+    server infra1 10.x.x.x:443 check
+
+# if you have multiple routers, you have to add all of them.
+#   server infra2 /10.x.x.y:443 check
```
Comment (sabre1041) [inline playbooks/operationalizing/cloudforms_networking.adoc]: Instead of multiple routers, do we want to say multiple nodes with routers deployed?

**Comment by sabre1041** [comment]
@Jooho Looks good. A couple minor issues. Is there supposed to be an example of balancing the pool of masters as well?

@etsauer @oybed please review as well

**Comment by sabre1041** [comment]
@Jooho bump for the question above

File: playbooks/operationalizing/cloudforms_networking.adoc (hunk 1 above)
Comment (Jooho) [inline playbooks/operationalizing/cloudforms_networking.adoc]: changed it

File: playbooks/operationalizing/cloudforms_networking.adoc (hunk 2 above)
Comment (Jooho) [inline playbooks/operationalizing/cloudforms_networking.adoc]: changed it

File: playbooks/operationalizing/cloudforms_networking.adoc (hunk 3 above)
Comment (Jooho) [inline playbooks/operationalizing/cloudforms_networking.adoc]: add the example configuration for masters.

**Comment by Jooho** [comment]
@sabre1041 I add the example configuration for masters.

**Pull request merged**